test/TestUtilities/Test.Utility/Commands/ProjectJsonTestHelpers.cs

```diff
@@ -186,5 +193,23 @@ private static PackageSpec WithProjectJsonTestRestoreMetadata(this PackageSpec s
 
             return updated;
         }
+
+        public static PackageSpec GetPackageSpec(string projectName, string rootPath = @"C:\", string framework = "net5.0")
```

Comment:
```cs
string rootPath = @"C:\"
```
I think the default setting could be a problem if we ever run tests cross platform. We are good for now.

Reply:
It's only a problem if you are using to run actual restore, which this isn't used for too often.

You are right however, it *will* be an obvious failure that should be easily addressable.